docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -147,19 +147,19 @@ Use the latest version of Prometheus or at least version 2.47.
        send_native_histograms: true
    ```
 
-## Scrape and send native histograms with Grafana Agent
+## Scrape and send native histograms with Grafana Alloy
```

Comment:
Could we please confirm these instructions?

Reply:
Seems correct to me minus the correction below from narqo.